daemon/k8s_watcher.go

```diff
@@ -1507,15 +1516,15 @@ func (d *Daemon) updateCiliumNetworkPolicyV2AnnotationsOnly(ciliumV2Store cache.
 
 	ctrlName := cnp.GetControllerName()
 
-	// Revision will *always* be populated because ruleRevCache is guaranteed
+	// Revision will *always* be populated because importMetadataCache is guaranteed
 	// to be updated by addCiliumNetworkPolicyV2 before calls to
 	// updateCiliumNetworkPolicyV2 are invoked.
-	rev, _ := ruleRevCache.get(cnp)
+	policyImportMetadata, _ := importMetadataCache.get(cnp)
 
 	k8sCM.UpdateController(ctrlName,
 		controller.ControllerParams{
 			DoFunc: func() error {
-				return cnpNodeStatusController(ciliumV2Store, cnp, rev, scopedLog, nil)
+				return cnpNodeStatusController(ciliumV2Store, cnp, policyImportMetadata.revision, scopedLog, policyImportMetadata.policyImportError)
```

Comment:
consider renaming `policyImportMetadata` to `metadata` as this line already has 149 columns